README.md

```diff
@@ -3,8 +3,255 @@
 [![Build Status](https://travis-ci.org/JuliaGeo/Geodesy.jl.svg?branch=master)](https://travis-ci.org/JuliaGeo/Geodesy.jl)
 [![Coverage Status](http://img.shields.io/coveralls/JuliaGeo/Geodesy.jl.svg)](https://coveralls.io/r/JuliaGeo/Geodesy.jl)
 
-Work with points defined in various coordinate systems.
+**Geodesy** is a Julia package for working with points in various world and
+local coordinate systems. The primary feature of *Geodesy* is to define and
+perform coordinate transformations in a convenient and safe framework,
+leveraging the *CoordinateTransformations* [package](https://github.com/FugroRoames/CoordinateTransformations.jl).
+Transformations are accurate and efficient and implemented in native Julia code
+(with many functions being ported from Charles Karney's *GeographicLib*
+[C++ library](http://geographiclib.sourceforge.net/)), and some common geodetic
+datums are provided for convenience.
 
-The code has been split out from [OpenStreetMap.jl](https://github.com/tedsteiner/OpenStreetMap.jl), and expanded slightly, but current functionality is basically limited to what is needed by OpenStreetMap.jl.
+### Quick start
 
-Coordinate systems `LLA`, `ECEF`, and `ENU` are supported. Most translations between between those types are supported, and use `WGS84` as the default datum. A couple other datums are provided, as well as an `Ellipsoid` type for defining datums. `distance` takes either two `ENU` or two `ECEF` points, and `Bounds{LLA}` and `Bounds{ENU}` are useful for operations such as `inBounds(point, bounds)`.
+Lets define a 3D point by it's latitude, longitude and altitude (LLA):
+```julia
+x_lla = LLA(-27.468937, 153.023628, 0.0) # City Hall, Brisbane, Australia
+```
+
+This can be converted to a Cartesian Earth-Centered-Earth-Fixed (ECEF)
+coordinate simply by calling the constructor
+```julia
+x_ecef = ECEF(x_lla, wgs84)
+```
+
+Here we have used the WGS-84 ellipsoid to calculate the transformation, but other
+datums such as `osgb36`, `nad27` and `grs80` are provided. All transformations
+use the *CoordinateTransformations*' interface, and the above is short for
+```julia
+x_ecef = transform(ECEFfromLLA(wgs84), x_lla)
+```
+where `ECEFfromLLA` is a type inheriting from *CoordinateTransformations*'
+`AbstractTransformation{ECEF,LLA}`. (Similar names `XfromY` exist for each of the
+coordinate types.)
+
+Often, points are measured or required in a *local* frame, such as the north-east-up
+coordinates with respect to a given origin. The `ENU` type represents this
+coordinate system and let's us do that simply to or from any globally referenced
+points.
```

Comment:
Confusing sentence.  (also let's -> lets)  How about

The `ENU` type represents points in this coordinate system and we may transform between ENU any globally referenced points.